src/lang/index.js

```diff
@@ -1,4 +1,75 @@
-// TODO read/write language setting to local cookie
-import en from './en.json';
+const savedLang = localStorage.getItem('localization');
+const lang = require(`./${savedLang || 'en'}.json`);
 
-export default en;
+export const langs = [
+  {
+    value: 'en',
+    en: 'English',
```

Comment:
think the "en" string should go in the string file as well?  If the user changes to e.g. Spanish, they should see "English - Ingles" or "Deutsch - Alemán"?